visualization/app/material/matMenu.scss

```diff
@@ -0,0 +1,16 @@
+.mat-menu-content {
+	.menu-header {
+		font-size: 9pt;
+		height: 20px;
+		line-height: 20px;
+	}
+
+	// overwrite global app.scss style
+	button {
+		margin: unset;
+	}
+}
+
+.mat-menu-item {
+	font-size: 16px;
```

Comment:
Generally id preffer pt over px, when working with fonts

Reply:
Generally I agree. But as we mix it up everywhere and have no concept for it so far, I don't want to invest any time into it while migrating.